packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -617,44 +644,35 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         uint256 requestTimestamp,
         bytes memory customAncillaryData
     ) private {
-        OptimisticOracleInterface optimisticOracle = _getOptimisticOracle();
-
         // Set reward to 0, since we'll settle proposer reward payouts directly from this contract after a relay
         // proposal has passed the challenge period.
-        optimisticOracle.requestPrice(
-            bridgeAdmin.identifier(),
-            requestTimestamp,
-            customAncillaryData,
-            IERC20(l1Token),
-            0
-        );
+        optimisticOracle.requestPrice(identifier, requestTimestamp, customAncillaryData, IERC20(l1Token), 0);
 
         // Set the Optimistic oracle liveness for the price request.
         optimisticOracle.setCustomLiveness(
-            bridgeAdmin.identifier(),
+            identifier,
             requestTimestamp,
             customAncillaryData,
-            uint256(bridgeAdmin.optimisticOracleLiveness())
+            uint256(optimisticOracleLiveness)
         );
 
         // Set the Optimistic oracle proposer bond for the price request.
         uint256 proposerBond = _getProposerBond(amount);
-        optimisticOracle.setBond(bridgeAdmin.identifier(), requestTimestamp, customAncillaryData, proposerBond);
+        optimisticOracle.setBond(identifier, requestTimestamp, customAncillaryData, proposerBond);
     }
 
     function _proposeOraclePriceRelay(
         uint256 amount,
         uint256 requestTimestamp,
         bytes memory customAncillaryData
     ) private {
-        OptimisticOracleInterface optimisticOracle = _getOptimisticOracle();
-        uint256 proposerBondPct =
-            FixedPoint.Unsigned(uint256(bridgeAdmin.proposerBondPct())).div(FixedPoint.fromUnscaledUint(1)).rawValue;
-        uint256 finalFee = _getStore().computeFinalFee(address(l1Token)).rawValue;
+        uint256 proposerBondPct_ =
```

Comment:
I like the underscore _ before the variable name, wdyt?

Reply:
sure, updated.